pkgdb2/api/packages.py

```diff
@@ -1105,9 +1139,9 @@ def api_package_critpath():
     return jsonout
 
 
-@API.route('/package/<package>/monitor/<status>', methods=['POST'])
+@API.route('/package/<namespace>/<package>/monitor/<status>', methods=['POST'])
```

Comment:
Sure.  I guess I thought it was possible with two ``@API.route(..`` decorators, but if not, then let's go for it.